File: domain/aplicativo/musicas/playlist.py
class Playlist:
    
    def __init__(self, nome, playlists):
        self.nome = nome
        self.playlists = playlists
        self.musicas = []
    
    def criar_playlist(cls, nome, usuario):
        playlist = Playlist(nome, usuario)
        return playlist
    
    
    def adicionar_musica(self, musica):
        self.musicas.append(musica)
        musica.registrar_playlist(self)
    
    
    def buscar_musicas(self, nome):
        return [musica.nome for musica in self.musicas if nome in musica.nome]

File: MusicApp/MusicApp.Domain/Aplicativo/Aggregates/Playlist.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MusicApp.Domain.Aplicativo.Aggregates {
    public class Playlist 
    {

        public Guid Id { get; set; }
        public string Nome { get; set; }
        public List<Musica> Musicas { get; set; }

        public Playlist() 
        {
            this.Musicas = new List<Musica>();
        }

        
        public void AdicionarMusica(Musica musica) { }

        public List <Musica> BuscarMusicas() { return null; }

    }
}
